src/dispatch/decorators.py

```diff
@@ -52,6 +55,7 @@ def _execute_task_in_project_context(
         )
     except Exception as e:
         # No rollback necessary as we only read from the database
+        log.error("ERROR trying to execute: " + fullname(func))
```

Comment:
f-strings are preferred
```suggestion
        log.error(f"Error trying to execute task: {fullname(func)}")
```